v4/internal/integrationsupport/integrationsupport_test.go

```diff
@@ -36,17 +36,19 @@ func TestSuccess(t *testing.T) {
 
 	app.ExpectSpanEvents(t, []internal.WantSpan{
 		{
-			Name:     "mySegment",
-			ParentID: internal.MatchAnyParent,
+			Name:          "mySegment",
+			ParentID:      internal.MatchAnyParent,
+			SkipAttrsTest: true,
 			Attributes: map[string]interface{}{
 				"parentId":                         internal.MatchAnything,
 				"category":                         "generic",
 				newrelic.SpanAttributeAWSOperation: "operation",
 			},
 		},
 		{
-			Name:     "hello",
-			ParentID: internal.MatchNoParent,
+			Name:          "hello",
+			ParentID:      internal.MatchNoParent,
+			SkipAttrsTest: true,
```

Comment:
It looks like we're setting `SkipAttrsTest` to `true` a lot. Would it make sense to change it to `IncludeAttrsTest` and have it default to false? Would that be less verbose?

Reply:
Wait, never mind, I didn't read this part:
> The field should be removed once all segment and transaction level attributes have been added.